presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,138 @@
+# Release 341 (XX Aug 2020)
+
+## General Changes
+
+* Add support for variable-precision `TIME` type. ({issue}`4381`)
+* Add Iceberg connector.
+* Add {func}`human_readable_seconds`. ({issue}`4344`)
+* Add support for {func}`extract` for `timestamp(p) with time zone` with values of `p` other than 3. ({issue}`4867`)
+* Add support for correlated subqueries in recursive queries. ({issue}`4877`)
+* Expose physical input bytes read statistics. ({issue}`4600`}
+* Report dynamic filters stats. ({issue}`4440`)
+* Improve query scalability when new nodes are added to cluster. ({issue}`4294`)
+* Reduce latency of how fast lazy dynamic filters are collected. ({issue}`4924`) 
+* Fix serialization of NULL values in structural types for old Presto clients ({issue}`4778`)
+* Fix failure when aggregation query contains duplicate expressions. ({issue}`4872`)
+* Fix compiler failure when querying long timestamp type. ({issue}`4824`)
+* Change timestamp operations to match the SQL specification. The value of a `TIMESTAMP` type is not affected by the session 
+  time zone. ({issue}`4799`)
+* Fix parsing failure of timestamps due to daylight saving changes. ({issue}`4799`)
+* Remove the `deprecated.legacy-timestamp` configuration property and the `legacy_timestamp` session property. ({issue}`4799`)
+* Preserve precision when applying `AT TIME ZONE` to values of type `TIMESTAMP`. ({issue}`4866`)
+* Fix failure when calling `EXTRACT` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `TIMESTAMP WITH TIME ZONE` type. ({issue}`4867`)
+* Fix query deadlock for connectors that wait for dynamic filters. ({issue}`4946`)
+* Add `optimizer_rule_stats` system table. ({issue}`4659`)
+* Fix failure when `TIME` or `TIMESTAMP` subtraction returns a negative value. ({issue}`4847`)
+* Fix failure when duplicate expressions appear in `DISTINCT` clause. ({issue}`4787`)
+
+## Security Changes
+
+* Add Salesforce password authentication. ({issue}`4372`)
+* Allow to configure refresh period for system level access control with environment variables. ({issue}`4854`)
+* Only request HTTPS client certificate when certificate authentication is enabled. ({issue}`4804`)
+
+## Web UI
+
+* Fix displaying physical input read time in detailed query view. ({issue}`4962`)
+
+## JDBC Driver Changes
+
+* Implement `ResultSet.getStatement()`. ({issue}`4957`)
```

Comment:
what does that do for the user?

Reply:
@mosabua I think it is just a completeness thing.  That method in the JDBC driver was not implemented and now it is